.github/workflows/pull_request.yml

```diff
@@ -1,11 +1,16 @@
 name: Pull request workflow
 
-on: pull_request
+on: push
```

Comment:
Should we rename this workflow file from `pull_request.yml` now that the trigger changed?